traveler-swift-core/TravelerKit/Models/PriceLimits.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  PriceLimits.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-08-20.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+/**
+ A model represeting a facet that relates to the maximum and minimum prices that items in a `CatalogItemSearchResult` may have.
+ */
+struct PriceLimits: Decodable {
+    /// The maximum price available for all the items in a `CatalogItemSearchResult`
+    public let max: Price
```

Comment:
Is this model only used for mapping?

Im very inclined to use PriceRange instead of this

Reply:
oh I see the challenge now

Reply:
Then I suggest

`PriceRange` -> `PriceRangeFilter`

Reply:
thoughts?